### PR TITLE
chore: use `semantic-release` cross-formula standard structure

### PR DESCRIPTION
**Pull request by myii**
* Automated using `ssf-formula` (v0.1.0-rc.1)

### CODE REVIEW

**Comment by myii** [comment]
@alxwr @n-rodriguez Any objections to me self-merging this?

File: kitchen.yml [hunk 1]
```diff
@@ -134,8 +116,21 @@ verifier:
...
-  inspec_tests:
-    - path: test/integration/default
 
 suites:
   - name: default
+    provisioner:
+      state_top:
+        base:
+          '*':
+            - openvpn.config
```
Comment (daks) [inline kitchen.yml]: So we don't test the metastate "openvpn"? only "openvpn.config"? There must have a reason. Anyway it was already like that so no impact on your PR.

**Comment by daks** [comment]
Merged for you.

File: kitchen.yml (hunk 1 above)
Comment (myii) [reply kitchen.yml]: Yes, noticed a few of the formulas doing this.  It's much easier to get an overview when all of the configuration elements are in the same file.  So we'll be able to investigate this a further as we make progress with all of this.

**Comment by myii** [comment]
Thanks, appreciate it.

**Pull request merged**